.gitignore

```
@@ -1,2 +1,3 @@
-venv/
+*venv/
 *.egg-info/
+*.log
```

```suggestion
*.log

```
Newline at end of file.